src/components/buttons/dropdown-buttons/styles.ts

```diff
@@ -33,7 +33,6 @@ export const DropdownButton = styled.div`
   &:hover {
     cursor: pointer;
     box-shadow: 0 0;
-    // TODO check color
```

Comment:
@pectom 
Szczerze mówiąc nie zrozumiałem tego `TODO`

Reply:
Chodzi o to, że niektóre kolory są w rgba, inne w hexie i możliwe, że mamy np. duplikaty

Reply:
Miałeś rację.
Ty był `primary`.
Podmieniłem i też wyniosłem inne kolory z rgba do stałych.